Added risk assessment to contributing page (#7831)
magento/devdocs: src/contributor-guide/contributing.md

The number of 'Yes' answers should help you to determine the severity.

### Pull request risk assessment:

> The risk label highlights the risk that the suggested changes may bring to the platform, it helps us to process pull requests, to decide on which version the pull requests should be delivered, it highlights pull requests that may need to be reviewed by other people, and it helps to decide whether a pull request should be approved or not depending on its risk.

Reviewer: Sentence is too long - whole paragraph is one sentence
```suggestion
The risk label highlights the risk that the suggested changes may bring to the platform. It helps the Document Maintainer Team to decide on which version the pull requests should be delivered, which may need to be reviewed by other people and whether a request should be approved or not depending on its risk.
```

Author: Hi @BarnyShergold, thanks for reviewing, I've partially applied the suggestion, I've replaced `Document Maintainer Team` with `maintainers` which seems to be more appropriate in this case.

| Risk | Description |
| ------------- | ------------- |

> | High | A pull request that makes changes on the framework or changes that will affect other areas. |

Reviewer: ```suggestion
| High | A pull request that makes changes on the framework or changes that will affect multiple areas. |
```

> | Medium | A pull request that makes changes that may affect other areas or it makes considerable changes on specific area. |

Reviewer: ```suggestion
| Medium | A pull request that makes changes which may affect multiple areas or makes considerable changes on a specific area. |
```

> | Low | A pull request that it's changes likely won't affect other areas. |

Reviewer: Should avoid contractions in technical documents - people with English as a second language may not understand.
```suggestion
| Low | A pull request that will probably not affect other areas. |
```
